src/Shell/Task/PluginTask.php

```diff
@@ -130,32 +123,23 @@ public function bake($plugin)
     }
 
     /**
-     * Update the app's bootstrap.php file.
+     * Modify the application class
      *
      * @param string $plugin Name of plugin
-     * @param bool $hasAutoloader Whether or not there is an autoloader configured for
      * the plugin
      * @return void
      */
-    protected function _modifyBootstrap($plugin, $hasAutoloader)
+    protected function _modifyApplication($plugin)
     {
-        $bootstrap = new File($this->bootstrap, false);
-        if (!$bootstrap->exists()) {
-            $this->err('<warning>Could not update application bootstrap.php file, as it could not be found.</warning>');
+        $application = new File(ROOT . DS . 'src' . DS . 'Application.php', false);
+        if (!$application->exists()) {
+            $this->err('<warning>Could not update application Application.php file, as it could not be found.</warning>');
 
             return;
         }
-        $contents = $bootstrap->read();
-        if (!preg_match("@\n\s*Plugin::loadAll@", $contents)) {
-            $autoload = $hasAutoloader ? null : "'autoload' => true, ";
-            $bootstrap->append(sprintf(
-                "\nPlugin::load('%s', [%s'bootstrap' => false, 'routes' => true]);\n",
-                $plugin,
-                $autoload
-            ));
-            $this->out('');
-            $this->out(sprintf('%s modified', $this->bootstrap));
-        }
+
+        $cmd = ['cake', 'plugin', 'load', $plugin];
+        ShellDispatcher::run($cmd);
```

Comment:
You can use `$this->runCommand()` instead.

Reply:
yep, makes sense, updated

Reply:
if you want to use `dispatchShell()`
remove the `use Cake\Console\ShellDispatcher;` line